File: exercises/D501/solution.cpp
#include <fstream>
#include <iostream>
using namespace std;
int main() {
    ifstream fin("account.txt");
    int n, x, balance = 0;
    fin >> n;
    for (int i = 0; i < n; i++) {
        fin >> x;
        balance += x;
    }
    fin.close();
    cout << balance << endl;
}

File: exercises/D501/solution.java
import java.io.File;
import java.io.FileNotFoundException;
import java.util.*;

class D501 {
    public static void main(String[] args) throws FileNotFoundException {
        File fin = new File("account.txt");
        Scanner in = new Scanner(fin);
        int n = in.nextInt();
        int balance = 0;
        for (int i = 0; i < n; i++) {
            int x = in.nextInt();
            balance += x;
        }
        in.close();
        System.out.println(balance);
    }
}
